course7/homeworks/anton/dc/complex_stack.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "complex.h"
+#include "stack"
+
+
+class ComplexStack
```

Comment:
Make sure that your classes ans functions do one thing, and one thing only.
What is this class doing? Is it a container? A command dispatcher? A parser?

Reply:
in terms of C# this class contains a bunch of extension methods for `std::stack<complex::Complex>`. I definitely don't like it. Would it be better if i implemented it as a set of functions taking `std::stack<complex::Complex>` as the first parameter? Formally, instead of one class breaking SRP, we would have a few functions compliant to SRP, but would it make any difference?